content/docs/2.14/operate/events.md

```diff
@@ -75,13 +78,40 @@ There will be multiple types of destination to emit KEDA events to.
 Here is an overview of the supported destinations:
 
 - [HTTP endpoint](#http-endpoint).
+- [Azure Event Grid endpoint](#azure-event-grid).
 
 #### HTTP endpoint
 ```yaml
   destination:
     http:
       uri: http://foo.bar  #An http endpoint that can receive cloudevent
 ```
+#### Azure Event Grid
+```yaml
+  destination:
+    azureEventGrid:
+      endpoint: Endpoint:foo.bar #endpoint from AzureEventGrid Topic
+```
+```TriggerAuthentication``` is needed to provide accesskey or identity for Azure Evnet Grid authentication.
```

Comment:
How do I link these two?

Reply:
KEDA will try to find if TA has a `key` value.

Reply:
That is not how it should work though, the end-user should specify the trigger authentication resource to use or there is a security risk